seed/challenges/01-front-end-development-certification/basic-bonfires.json

```diff
@@ -37,7 +37,34 @@
       "tests": [],
       "type": "Waypoint",
       "challengeType": 7,
-      "nameEs": "Prepárate para los Ziplines",
+      "nameEs": "Prepárate para nuestros Desafíos sobre Algoritmos",
+      "descriptionEs":  [
+        [
+          "http://i.imgur.com/sJkp30a.png",
+          "Una imagen de un desafio sobre algoritmos que presenta instrucciones, pruebas y el editor de código.",
+          "Nuestros desafios sobre algoritmos te enseñarán como pensar como un programador.",
+          ""
+        ],
+        [
+          "http://i.imgur.com/d8LuRNh.png",
+          "Una mamá pájaro saca un bebé pájaro fuer de su nido.",
+          "Nuestros desafios anteriores te introdujeron a los conceptos de programación. Pero para estos desafios sobre algoritmos, ahora necesitarás aplicar lo que has aprendido y resolver problemas de respuesta abierta",
+          ""
+        ],
+        [
+          "http://i.imgur.com/WBetuBa.jpg",
+          "Un programador frustado golpeando la pantalla de su computador.",
+          "Nuestros desafíos sobre algortimos son difíciles.  Algunos pueden requerir muchas horas para resolverse.  Usted quedará frustado.  Pero no se rinda.",
```

Comment:
For consistency, change `Usted quedará frustado.  Pero no se rinda.` for something along the lines of `Podrás frustrarte, pero no te rindas.`